detemplatize.js

```diff
@@ -11,9 +11,10 @@ module.exports = function (text) {
   // {% anything %} is replaced with a /* ... */
   // {% raw xxx %} is replaced with a {/* ... */} -- this is typically assigned to a variable
   // {{ anything }} is replaced with a {/* ... */} -- this is typically assigned to a variable
+  // DJANGO templates are ALWAYS onelined
   return text
-    .replace(/{#[\s\S]*?#}/g, replace_with_comments)
-    .replace(/{%\s*raw\s[\s\S]*?%}/g, replace_with_obj)
-    .replace(/{%[\s\S]*?%}/g, replace_with_comments)
-    .replace(/{{[\s\S]*?}}/g, replace_with_obj)
+    .replace(/{#.*?#}/g, replace_with_comments)
```

Comment:
Will not match multi-line templates

Reply:
Yes indeed :) see https://code.djangoproject.com/ticket/8652